frontend/components/commons/results/PaginationEndAlert.vue

```diff
@@ -1,12 +1,30 @@
 <template>
   <div class="results-pagination-end">
     <svgicon width="40" height="40" name="row-last" />
-    <p>
-      You have reached the current record limit ({{ limit | formatNumber }}). To
-      view more records please refine your query
-    </p>
+    <div class="desc">
+      <p>
+        You cannot go through more than {{ limit | formatNumber }} records. If
+        you want to explore more records, you can combine queries, filters, and
+        sorting to fine-grain your search. For more info about queries, visit
+        the
+        <a
+          href="https://docs.argilla.io/en/latest/guides/features/queries.html?highlight=queries#"
+          target="_blank"
+        >
+          docs</a
+        >. If you want, you can configure this limit at the cost of potential
+        performance loss. For more details about this limit, see
+        <a
+          href="https://docs.argilla.io/en/latest/reference/webapp/pages.html#dataset"
+          target="_blank"
+        >
```

Comment:
```suggestion
      <p>
        You cannot go through more than {{ limit | formatNumber }} records. If
        you want to explore more records, you can combine queries, filters, and
        sorting to fine-grain your search. For more info about queries, visit
        the
        <a
          href="https://docs.argilla.io/en/latest/guides/features/queries.html?highlight=queries#"
          target="_blank"
        >
          docs</a
        >. For now, changing this limit is not possible. For more details about this limit, visit 
        the official es docs
        <a
          href="https://www.elastic.co/guide/en/elasticsearch/reference/current/paginate-search-results.html#paginate-search-results"
          target="_blank"
        >
```

Reply:
Since we cannot change this limit for now, I propose to inform the users and reference the official elasticsearch documentation for more details.

cc @davidberenstein1957 @dvsrepo 

Ref: #2239